client/app/telecom/pack/xdsl/modem/pack-xdsl-modem.html

```diff
@@ -129,6 +128,13 @@ <h2 class="widget-presentation-title" data-translate="xdsl_modem_advanced_config
                         </div>
                     </div>
                 </div>
+
+                <div class="my-3" data-ng-class="{ 'disabled' : !XdslModem.mediator.capabilities.canChangeFirmware || XdslModem.mediator.tasks.changeModemConfigFirmware }">
+                    <label for="configFirmware" oui-tooltip="{{'xdsl_modem_firmware_description' | translate}}" 
```

Comment:
There is no labelable content with the `cnfigFirmware` id.

alors `data-oui-tooltip="{{:: 'xdsl_modem_firmware_description' | translate }}"`